scripts/run_endpoint.sh

```diff
@@ -41,3 +55,5 @@ elif [ "$ROLE" == "server" ]; then
     quicinteropserver ${SERVER_PARAMS} -root:/www -listen:* -port:443 \
         -file:/server.crt -key:/server.key 2>&1
 fi
+
+trap finish EXIT
```

Comment:
Ok. Well the logs start just fine, but it seems this `trap` isn't working as expected. I never get any output from the server when it's shutdown.